src/figuresClasses.ts

```diff
@@ -1,11 +1,92 @@
-export interface Figure {}
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
+
+  constructor(
+    public color: Color,
+    public firstSide: number,
+    public secondSide: number,
+    public thirdSide: number,
```

Comment:
According to the checklist, when declaring constructor parameters with access modifiers, TypeScript implicitly creates properties on the class. You don't need to assign them manually. Remove the redundant assignments to avoid duplicating the work that TypeScript does for you.